terraform/variables.tf

```diff
@@ -1,13 +1,19 @@
 variable "image_path" {
   type        = string
   description = "Where the images will be stored"
-  default     = "/code/ubuntu-pool"
+  default     = "/home/user/images"
+}
+
+variable "private_key_path" {
+  type        = string
+  description = "SSH private key path"
+  default     = "/home/user/.ssh/k8s_rsa"
```

Comment:
I'd remove the "default" here @jkryl --then terraform wil prompt you instead of going off with a known to be broken default.

Reply:
Oh, and the default is wrong. I left the path from my env 🤦  Thanks! will fix

Reply:
just an idea for all defaults, what about leaving the reasonably sensible defaults but commented out? so we'd know what to edit should anyone want to "hardcode" them?

Reply:
good idea 👍